packages/test/test-end-to-end-tests/src/test/compression.spec.ts

```diff
@@ -85,13 +96,65 @@ const compressionSuite = (getProvider) => {
 			if (provider.type === "TestObjectProviderWithVersionedLoad") {
 				this.skip();
 			}
+			await setupContainers();
 			const value = generateRandomStringOfSize(5);
 			localMap.set("testKey", value);
 
 			await provider.ensureSynchronized();
 			assert.strictEqual(localMap.get("testKey"), value);
 			assert.strictEqual(remoteMap.get("testKey"), value);
 		});
+
+		const messageGenerationOptions = generatePairwiseOptions<{
+			compressionAndChunking?: {
+				chunking: boolean;
+			};
+			grouping: boolean;
+		}>({
+			compressionAndChunking: [undefined, { chunking: false }, { chunking: true }],
```

Comment:
Quick commenting explaining what `undefined` does, and what Compression does in these three cases?

Reply:
Like, should these tests run in FullCompat mode?